### ISSUE TITLE
What's the difference between data contamination and model contamination?

### ISSUE THREAD

**Issue by JaydencoolCC** [ntunlp/LLMSanitize#6]
To the best of my knowledge, contamination means there is an overlap between the training set and the test set. What's the difference between data contamination and model contamination? Can you help me explain it? thanks!

**Comment by Ravoxsg** [comment]
Hi @JaydencoolCC, thank your for your interest in our work!

You are right that the core concept of contamination is _prior exposure of the LLM to the test set_, aka an _overlap between the training set and the test set(s)._ 
Here we made this distinction between data and model contamination depending on _**whether we have access to the training set**._ 
- If we have access to the training set, then it is a **data contamination** use case: we simply compute the overlap between both sets (e.g. n-gram matching, embeddings similarity, etc). We don't need the LLM. We directly work with the datasets. 
- If we do not have access to the training set (which is often the case with many LLMs! An even if we do have access, it could be very large and thus very hard to work with), then we only have access to the LLM. In that case it is a **model contamination** use case. The methods involved will focus on the LLM's behavior: (over)confidence, capability to exactly complete an example when prompted with the first few words, etc. Such methods are quite different compared to methods simply comparing data points to each other. 

Does it make sense?

**Comment by JaydencoolCC** [comment]
Thank you very much for your reply!  I fully understand it. @Ravoxsg

**Comment by Ravoxsg** [comment]
Cool, I am glad I could help!
